src/lang/pl.json

```diff
@@ -361,6 +361,7 @@
     "serwersmsAPIUser": "Nazwa użytkownika API (z prefiksem webapi_)",
     "serwersmsAPIPassword": "Hasło API",
     "serwersmsPhoneNumber": "Numer telefonu",
+    "serwersmsPhoneNumberGroupId": "Numer telefonu lub ID Grupy",
```

Comment:
We still value these translations, but they need to be via weblate.
Only `en.json` can/should be translated to make merge conflicts with weblate less likely.
If new translation keys are not avaliable in `en.json`, our translators cannot translate these keys.